src/components/editPostModal/index.js

```diff
@@ -35,8 +36,31 @@ const EditPostModal = () => {
     setIsEditing(false);
   };
 
-  const handleConfirmDelete = () => {
-    handleActionWithMessage("Post deleted! Closing modal in 2 seconds...");
+  const handleConfirmDelete = async (postId) => {
+    try {
+      await deletePost(postId);
+      handleActionWithMessage("Post deleted! Closing modal in 2 seconds...");
```

Comment:
This text is incorrect. You're not supplying the `timeout` variable, so the default time is actually 5 seconds. Not 2.